hr_attendance_rfid/data/ir_rule.xml

```diff
@@ -0,0 +1,15 @@
+<odoo>
```

Comment:
```suggestion
<odoo noupdate="1">
```

Reply:
If we are adding the noupdate, we should need to add a migration script that will create the rule

Reply:
Really? even if that `ir.model.data` record is not already set on the system? OUCH

Reply:
Or the forcecreate, but I would opt for the migration

Reply:
Something as hard as this:
https://github.com/OCA/community-data-files/blob/12.0/account_payment_unece/migrations/12.0.1.0.0/post-migration.py#L8 :wink:

Reply:
I don't know about migration script.
if you add `noupdate` in code, Will it cause problems?

Reply:
> Something as hard as this:
> https://github.com/OCA/community-data-files/blob/12.0/account_payment_unece/migrations/12.0.1.0.0/post-migration.py#L8 wink

I think this is the best option, better than the force_create